src/sentry/static/sentry/app/components/checkboxFancy.tsx

```diff
@@ -5,6 +5,13 @@ import {css} from '@emotion/core';
 import PropTypes from 'prop-types';
 import InlineSvg from 'app/components/inlineSvg';
 
+type Props = {
+  disabled?: boolean;
+  checked?: boolean;
+  size?: string;
+  className?: string;
+} & HTMLDivElement;
```

Comment:
Is this right? I think it's like, `React.HTMLAttributes<HTMLDivElement>` or something like that